src/webgpu/shader/execution/expression/call/builtin/select.spec.ts

```diff
@@ -133,7 +140,7 @@ g.test('scalar')
 
     await run(
       t,
-      builtin('select'),
+      t.params.component !== 'af' ? builtin('select') : abstractBuiltin('select'),
```

Comment:
nit.  I would flip the condition around and rephrase it in the positive sense.

Reply:
there's a similar one later too.

not a blocking request

Reply:
Done